server/package.json

```
@@ -39,20 +39,10 @@
     "db:migration:check": "node --loader ts-node/esm scripts/migration_check.ts"
   },
   "dependencies": {
-<<<<<<< HEAD
-    @apollo/server: "5.4.0",
-=======
-<<<<<<< HEAD
+    "@apollo/server": "5.4.0",
```

Biome (2.4.6)

[error] The key `@apollo/server` was already declared.

(lint/suspicious/noDuplicateObjectKeys)